applications/ironside/se/update/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _ironside_se_update_application:
+
+IRONside SE Update
+##################
+
+The *IRONside SE Update* application updates the IRONside SE firmware on a SoC that already has IRONside SE installed.
+It can update both the main image and the recovery image of IRONside SE using the IRONside SE firmware release ZIP file.
+
+Update procedure
+****************
+
+The application invokes the IRONside SE update service and passes the parameters that correspond to the location of the HEX file of the IRONside SE firmware update.
+   
+The application prints the return value of the service call and outputs information from the update HEX file.
+
+After the service call completes, the IRONside SE firmware updates the internal state of the device.
+The firmware installs the update during the next device boot.
+
+Requirements
+************
+
+The firmware supports the following development kit:
+
+.. table-from-sample-yaml::
+
+
+Building and running
+********************
+
+This application can only be used when there is already a version of IRONside SE installed on the device.
+To build and run the application, do the following:
+
+1. Unzip the update release ZIP to get the update HEX file.
+
+   .. code-block:: console
+      unzip nrf54h20_soc_binaries_v.*.zip
+     
+#. Build and program this application
+
+   .. code-block:: console
+      west build -b nrf54h20dk/nrf54h20/cpuapp/iron
+      west flash
+     
+#. Program the appropriate update hex file:
+
+   .. code-block:: console
+      # Perform one of the following commands
+      nrfutil device program --traits jlink --firmware update/ironside_se_update.hex # to update IRONside SE firmware
+      nrfutil device program --traits jlink --firmware update/ironside_se_recovery_update.hex # to update IRONside SE recovery firmware
+     
+#. Trigger a reset
+
+   .. code-block:: console
+      # Perform one of the following commands
+      nrfutil device reset --reset-kind RESET_PIN --traits jlink
+
```

Comment:
potentially add steps to wait + verify via boot-report version that the update worked